firefox-ios/Client/Frontend/Home/Homepage Rebuild/Redux/HomepageState.swift

```diff
@@ -17,6 +17,17 @@ struct HomepageState: ScreenState, Equatable {
     let pocketState: PocketState
     let wallpaperState: WallpaperState
 
+    /// FXIOS-11504 - This is mainly used for telemetry for top sites and pocket and presenting CFRs.
+    /// At this time, we are keeping `isZeroSearch` the same as legacy. However, we should revisit this value
+    /// and confirm what the expectation is, as it seems inconsistent. See more details in ticket.
+    ///
+    /// FXIOS-6203 - Comment from legacy homepage:
+    /// `isZeroSearch` is true when the homepage is created from the tab tray, a long press
+    /// on the tab bar to open a new tab or by pressing the home page button on the tab bar.
+    /// The zero search page, aka when the home page is shown by clicking the url bar from a loaded web page.
+    /// This needs to be set properly for telemetry and the contextual pop overs that appears on homepage
```

Comment:
cc: @lmarceau @ih-codes since I chatted with you both briefly on the historical context and telemetry